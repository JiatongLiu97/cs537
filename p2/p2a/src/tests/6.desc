getnumsyscalls test: Iterate through the process table.
