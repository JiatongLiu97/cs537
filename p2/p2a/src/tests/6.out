XV6_TEST_OUTPUT PID:1 21
XV6_TEST_OUTPUT PID:2 11
XV6_TEST_OUTPUT PID:3 50
XV6_TEST_OUTPUT PID:4 -1
XV6_TEST_OUTPUT PID:5 -1
XV6_TEST_OUTPUT PID:6 -1
XV6_TEST_OUTPUT PID:7 -1
XV6_TEST_OUTPUT PID:8 -1
XV6_TEST_OUTPUT PID:9 -1
XV6_TEST_OUTPUT PID:10 -1
XV6_TEST_OUTPUT PID:11 -1
XV6_TEST_OUTPUT PID:12 -1
XV6_TEST_OUTPUT PID:13 -1
XV6_TEST_OUTPUT PID:14 -1
XV6_TEST_OUTPUT PID:15 -1
XV6_TEST_OUTPUT PID:16 -1
XV6_TEST_OUTPUT PID:17 -1
XV6_TEST_OUTPUT PID:18 -1
XV6_TEST_OUTPUT PID:19 -1
XV6_TEST_OUTPUT PID:20 -1
XV6_TEST_OUTPUT PID:21 -1
XV6_TEST_OUTPUT PID:22 -1
XV6_TEST_OUTPUT PID:23 -1
XV6_TEST_OUTPUT PID:24 -1
XV6_TEST_OUTPUT PID:25 -1
XV6_TEST_OUTPUT PID:26 -1
XV6_TEST_OUTPUT PID:27 -1
XV6_TEST_OUTPUT PID:28 -1
XV6_TEST_OUTPUT PID:29 -1
XV6_TEST_OUTPUT PID:30 -1
XV6_TEST_OUTPUT PID:31 -1
XV6_TEST_OUTPUT PID:32 -1
XV6_TEST_OUTPUT PID:33 -1
XV6_TEST_OUTPUT PID:34 -1
XV6_TEST_OUTPUT PID:35 -1
XV6_TEST_OUTPUT PID:36 -1
XV6_TEST_OUTPUT PID:37 -1
XV6_TEST_OUTPUT PID:38 -1
XV6_TEST_OUTPUT PID:39 -1
XV6_TEST_OUTPUT PID:40 -1
XV6_TEST_OUTPUT PID:41 -1
XV6_TEST_OUTPUT PID:42 -1
XV6_TEST_OUTPUT PID:43 -1
XV6_TEST_OUTPUT PID:44 -1
XV6_TEST_OUTPUT PID:45 -1
XV6_TEST_OUTPUT PID:46 -1
XV6_TEST_OUTPUT PID:47 -1
XV6_TEST_OUTPUT PID:48 -1
XV6_TEST_OUTPUT PID:49 -1
XV6_TEST_OUTPUT PID:50 -1
XV6_TEST_OUTPUT PID:51 -1
XV6_TEST_OUTPUT PID:52 -1
XV6_TEST_OUTPUT PID:53 -1
XV6_TEST_OUTPUT PID:54 -1
XV6_TEST_OUTPUT PID:55 -1
XV6_TEST_OUTPUT PID:56 -1
XV6_TEST_OUTPUT PID:57 -1
XV6_TEST_OUTPUT PID:58 -1
XV6_TEST_OUTPUT PID:59 -1
XV6_TEST_OUTPUT PID:60 -1
XV6_TEST_OUTPUT PID:61 -1
XV6_TEST_OUTPUT PID:62 -1
XV6_TEST_OUTPUT PID:63 -1
